modules/distribution/keeper/querier.go

```diff
@@ -0,0 +1,242 @@
+package keeper
+
+import (
+	"github.com/irisnet/irishub/codec"
+	"github.com/irisnet/irishub/modules/distribution/types"
+	sdk "github.com/irisnet/irishub/types"
+	abci "github.com/tendermint/tendermint/abci/types"
+	"github.com/tendermint/tendermint/libs/log"
+)
+
+// nolint
+const (
+	QueryWithdrawAddr          = "withdraw_addr"
+	QueryDelegationDistInfo    = "delegation_dist_info"
+	QueryAllDelegationDistInfo = "all_delegation_dist_info"
+	QueryValidatorDistInfo     = "validator_dist_info"
```

Comment:
```
	QueryDelegationDistInfo    = "delegation_dist_info"
	QueryAllDelegationDistInfo = "all_delegation_dist_info"
	QueryValidatorDistInfo     = "validator_dist_info"
```
The above three querier doesn't make too much sense. It is not necessary to implement them

Reply:
Will decide which APIs are supported in the future, but not now.